### DISCUSSION TITLE
How to write a script to open multiple tabs and execute commands

### DISCUSSION THREAD

**Discussion by ahmed-adly-khalil** [vercel/hyper#6088] [Q&A]
Hello, 
Is there a way to write a shell script to open multiple tabs with multiple windows within each tab and execute commands to run different applications within these windows / tabs? 

Thank you

**Comment by LabhanshAgrawal** [comment]
Right now, we don't have such capability built in (there might be some plugins trying to do this)
I'm planning some additions to the cli but it'll take some time, comments and suggestions are always welcome.

**Reply by ahmed-adly-khalil** [reply]
Awesome, thanks
